src/libs/ajax/AzureStorage.js

```diff
@@ -130,12 +130,12 @@ export const AzureStorage = signal => ({
         ).then(res => res.text())
       },
 
-      create: async textContents => {
+      create: async contents => {
         const azureStorageUrl = await getBlobUrl(workspaceId, blobName)
 
         return fetchOk(azureStorageUrl, {
-          method: 'PUT', body: JSON.stringify(textContents),
-          headers: { 'Content-Type': 'application/x-ipynb+json', 'x-ms-blob-type': 'BlockBlob', 'Content-Length': _.size(textContents) }
+          method: 'PUT', body: contents,
+          headers: { 'Content-Type': 'application/x-ipynb+json', 'x-ms-blob-type': 'BlockBlob', 'Content-Length': contents.size }
```

Comment:
Does the `Content-Length` header need to be set manually? Does fetch not handle this for us?

Reply:
Let me try without. 
A different create function uses content length, so I thought I'd add it.

Reply:
b9430bdce705f97b6d9b4a0ba12da477366beb0a removed `Content-Length` from the GoogleStorage version of this. Is it needed here?

Reply:
Nope! good catch